src/vector.py

```diff
@@ -4,19 +4,80 @@ class Vector:
   """
   The vector class will create a vector object from a calculated value. It will
   create magnitude, direction, and component attributes.
+
+  Attributes:
+    x : float
+      value of x in rectangular coordinate system
+    y : float
+      value of x in rectangular coordinate system
+    r: float
+      value of radius in polar coordinate system
+    theta : float
+      value of theta (in radians) in polar coordinate system
   """
 
-  def __init__(self, x, y):
+  def __init__(self, *, x=None, y=None, r=None, theta=None):
+    """
+    Initializes a vector from either rectangular or polar coordinate representation.
```

Comment:
Let's try to keep our line length maxed out around 80 characters

Reply:
Adjusted to 79 characters, per PEP8 https://peps.python.org/pep-0008/#maximum-line-length